src/ledgerExecutor.ts

```diff
@@ -198,14 +198,28 @@ export class LedgerExecutor implements ILedgerExecutor {
     if (!ethereumFeeService) {
       throw new Error(`Ethereum Fee Service is not available.`);
     }
+
     // To fix this awkward comparison, make the comit_node use chainId for Ethereum: https://github.com/comit-network/RFCs/issues/73
-    if (
-      !(network === "regtest" && (await ethereumWallet.getChainId()) === 17)
-    ) {
+    const ethereumNetworkMappings: { [network: string]: number | undefined } = {
+      regtest: 17,
+      ropsten: 3
+    };
+
+    const mappedChainId = ethereumNetworkMappings[network];
+
+    if (!mappedChainId) {
```

Comment:
❓
Since we had issues with error logging, should we just log this error here as well?